gui/wxpython/location_wizard/wizard.py

```diff
@@ -2659,13 +2636,12 @@ def OnWizFinished(self):
                                       filename=self.filepage.georeffile,
                                       desc=self.startpage.locTitle)
             elif coordsys == "wkt":
-                if not self.wktpage.wktfile or \
-                        not os.path.isfile(self.wktpage.wktfile):
-                    return _("File <%s> not found." % self.wktpage.wktfile)
+                if not self.wktpage.wktstring:
+                    return _('WKT string missing.')
 
                 grass.create_location(dbase=self.startpage.grassdatabase,
                                       location=self.startpage.location,
-                                      wkt=self.wktpage.wktfile,
+                                      wkt=self.wktpage.wktstring,
```

Comment:
The error you are getting comes from this function, it expects file not string. I created a separate PR #723, so you don't need to deal with that.